### PR TITLE
My followers following search screens

### PR DESCRIPTION
**Pull request by prastoin**
close https://github.com/AdonisEnProvence/MusicRoom/issues/428
close https://github.com/AdonisEnProvence/MusicRoom/issues/429

### CODE REVIEW

File: packages/client/screens/MyProfile/MyFollowers/index.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+const MyFollowersSearchScreen: React.FC<MyFollowersScreenProps> = ({
+    navigation,
+}) => {
+    const insets = useSafeAreaInsets();
+    const sx = useSx();
+    const [screenOffsetY, setScreenOffsetY] = useState(0);
+    const initialNumberOfItemsToRender = IS_TEST ? Infinity : 10;
+
+    const [userFollowersSearchState, userFollowersSearchMachineSend] =
+        useMachine(() => myFollowerSearchMachine);
```
Comment (Devessier) [inline packages/client/screens/MyProfile/MyFollowers/index.tsx:27]: **💭 thought:** ‏As `myFollowerSearchMachine` remains always the same, it's not necessary to use a function here.

File: packages/client/screens/MyProfile/MyFollowing/index.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+const MyFollowingSearchScreen: React.FC<MyFollowingScreenProps> = ({
+    navigation,
+}) => {
+    const insets = useSafeAreaInsets();
+    const sx = useSx();
+    const [screenOffsetY, setScreenOffsetY] = useState(0);
+    const initialNumberOfItemsToRender = IS_TEST ? Infinity : 10;
+
+    const [userFollowingSearchState, userFollowingSearchMachineSend] =
+        useMachine(() => myFollowingSearchMachine);
```
Comment (Devessier) [inline packages/client/screens/MyProfile/MyFollowing/index.tsx:27]: **💭 thought:** ‏As `myFollowingSearchMachine` always remains the same, it's not necessary to use a function here.

File: packages/client/screens/MyProfile/MyFollowing/index.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                                              }}
+                                          >
+                                              Load more
+                                          </Text>
+                                      </TouchableOpacity>
+                                  </View>
+                              );
+                          }
+                        : undefined
+                }
```
Comment (Devessier) [inline packages/client/screens/MyProfile/MyFollowing/index.tsx:155]: **💭 thought:** ‏I recently discovered that `ListFooterComponent` prop accepts either a React function (as here) or a React component. The advantage of using a React component here is that React would be able to cache the Footer component. As it is written currently, a new component is created for each render, and I think that React is unmounting it for each render. This is an interesting thing that I want to deepen 👀.

See [React Native documentation for FlatList as reference](https://reactnative.dev/docs/flatlist#listfootercomponent)

**Review by Devessier — DISMISSED** [review]
Nice job!
I like that you think your tests in term of _models_. 👍 
I wrote some thoughts, we can merge.

File: packages/client/screens/MyProfile/MyFollowing/index.tsx (hunk 3 above)
Comment (prastoin) [reply packages/client/screens/MyProfile/MyFollowing/index.tsx:155]: Interesting :open_mouth:

**Review by Devessier — APPROVED** [review]
Great job!
We can merge.

**Pull request merged**